thirdparties/memcache/Makefile

```diff
@@ -2,7 +2,7 @@
 .PHONY: download build clean
 
 build: clean download
-	@cd libmemcached-1.1.2 && mkdir build-libmemcached && cd build-libmemcached && cmake .. && make libmemcached
+	@cd libmemcached-1.1.2 && mkdir build-libmemcached && cd build-libmemcached && cmake .. && make libmemcached -j$$(nproc)
```

Comment:
extra `$` ?
```suggestion
	@cd libmemcached-1.1.2 && mkdir build-libmemcached && cd build-libmemcached && cmake .. && make libmemcached -j$(nproc)
```

Reply:
I think we need this extra `$`.

Without the extra `$`, it becomes `$(nproc)`, which is a make variable, which is empty.

With the `$`, it escapes `$`, `$$(nproc)` becomes a shell command to get the number of logical processors, which is equal to `$(shell nproc)`

Reply:
well done!